day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,65 @@
+# Three little monkeys jumping on the bed,
```

Comment:
Great job on these 2 day-7 challenges Laura! I love seeing that you broke up your concerns into a method and a class. If you have time to keep working this weekend, then I encourage you to attempt the other two challenges. You can time box these to get practice at working under a time constraint! You're looking strong coming into mod-1! Good luck!